bodhi/server/scripts/check_policies.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright ® 2017 Red Hat, Inc.
```

Comment:
This should be a © instead of a ®. I see that I made that mistake in some of my headers, so I've filed https://github.com/fedora-infra/bodhi/pull/1695 to fix that. Sorry!